Quaver.Shared/Screens/Importing/ImportingScreenView.cs

```diff
@@ -121,7 +120,7 @@ private void HandleLoadingWheelAnimations()
         /// </summary>
         /// <param name="sender"></param>
         /// <param name="e"></param>
-        private void OnFinishedImporting(object sender, ImportingMapsetEventArgs e) => InformationText.Text = $"Now importing {e.FileName}. {e.Index}/{e.Queue.Count}";
+        private void OnFinishedImporting(object sender, ImportingMapsetEventArgs e) => InformationText.Text = $"Now importing {e.FileName}. {e.Index + 1}/{e.Queue.Count}";
```

Comment:
Rename the method to `OnImporting`, since the event is invoked before the actual import begins - not after it completes.

Reply:
renamed to OnImportingMapset to make consistent with method in mapsetImporter